### ISSUE TITLE
La pagina "Le persone" non mostra più anche le strutture

### ISSUE THREAD

**Issue by Simsimus** [italia/design-scuole-wordpress-theme#552]
### Prerequisiti

- [X] Ho già [cercato](https://github.com/italia/design-scuole-wordpress-theme/issues?utf8=%E2%9C%93&q=is%3Aissue) tra le issue la presenza di un problema simile
- [X] Ho letto le [linee guida di contribuzione](https://github.com/italia/design-scuole-wordpress-theme/blob/main/CONTRIBUTING.md)

### Descrivi il problema

Buonasera, 
dalla versione 3.7.0, nella pagina "Le persone" non vengono più mostrate le strutture impostate nella Configurazione > Le persone, a meno che non sia presente un utente associato ad una delle strutture selezionate.
Credo che questo sia un problema in diverse scuole che spesso vogliono elencare le persone attraverso le strutture, ma non sempre tali persone devono corrispondere ad un utente registrato nel sito.
In pratica potrebbero voler registrare, ad esempio, la struttura Segreteria inserendo nella scheda i nomi del personale amministrativo, ma non necessariamente registrare un account per ognuno di loro.
Il risultato che si otteneva fino alla versione 3.6.0 soddisfaceva meglio le scuole che avevano la doppia possibilità.
Cosa ne pensate?

### Su quale sisitema operativo stai riscontrando il problema?

Windows

### Su quale browser stai riscontrando il problema?

Qualsiasi

### Che versione del tema stai utilizzando?

3.7.0

**Comment by vcoopcsc** [comment]
La modifica è stata fatta in linea con il documento di architettura dell'informazione, che prevede che la pagina "Le persone" abbia all'interno 3 categorie: Dirigenza scolastica, personale docente e personale non docente.

**Comment by Simsimus** [comment]
E questo va bene, ma per inserire la dirigenza scolastica, il personale docente ed il personale non docente ora è necessario inserire ogni singolo utente con relativa user e password, mentre prima era possibile inserire le tre strutture all'interno delle quali, a livello descrittivo, potevano essere riportati tutti i nomi necessari senza dovergli attivare un account.

**Comment by sviluppoRobyone** [comment]
Ma la domanda da farsi e fare all'AGID sarebbe se è conforme GDPR mostrare tutte le schede del personale

**Comment by vcoopcsc** [comment]
Per il GDPR dipende da come viene gestita, per le scuole interessate a far vedere le persone abbiamo consigliato di fare un provvedimento del dirigente scolastico che funge da base giuridica per la pubblicazione dei dati personali, altri chiedono il consenso ma per le PA non è una base giuridica generalmente utilizzabile. In generale, è l'RPD dell'istituto che deve consigliare il metodo più idoneo. Per la figura del DS, invece, la cosa non rileva dato che è anche un obbligo previsto dal decreto trasparenza.

**Comment by vcoopcsc** [comment]
In ogni caso per non far comparire le persone in quella pagina dovrebbe bastare modificare il ruolo dell'utente nella sezione "Utenti" togliendo la voce Personale docente/ATA/Dirigente scolastico

**Comment by sviluppoRobyone** [comment]
Condivido che quella pagina sia il punto di partenza, quello che non è chiaro è:
1) se effettivamente c'è questa divisione per strutture organizzative (negli schemi dell'architettura non sembra esserci menzione)
2) si sì, quali strutture mostrare: le scuole? gli organi collegiali?

**Comment by vcoopcsc** [comment]
@sviluppoRobyone esatto, interpretando in maniera stringente il documento di architettura sembra che la pagina debba essere divisa in 3 macro-sezioni: Dirigente, personale docente e personale non docente e che ad ognuna di queste 3 sezioni vadano agganciate le persone.
